addons/project/models/project.py

```diff
@@ -1072,6 +1085,9 @@ def create(self, vals_list):
                         default_project_id=project_id
                     ).default_get(['stage_id']).get('stage_id')
                 vals["stage_id"] = default_stage[project_id]
+            if "project_id" in vals and "informal_project_id" not in vals and\
+               ("parent_id" not in vals or not vals.get('parent_id')):
```

Comment:
Actually dict.get returns ``None`` is the key is not present. Similar to 
```suggestion
            if "project_id" in vals and "informal_project_id" not in vals and not vals.get('parent_id')):
```